main.go

```diff
@@ -28,6 +43,72 @@ func createIntCounter(meter metric.Meter, name string, description string) metri
 		)
 }
 
+func createTracesAndSpans(ctx context.Context, tracesProvides *sdktrace.TracerProvider, suites []junit.Suite) error {
+	tracer := tracesProvides.Tracer("junit2otlp")
+	meter := global.Meter("junit2otlp")
+
+	durationCounter := createIntCounter(meter, TestsDuration, "Duration of the tests")
+	errorCounter := createIntCounter(meter, ErrorTestsCount, "Total number of failed tests")
+	failedCounter := createIntCounter(meter, FailedTestsCount, "Total number of failed tests")
+	passedCounter := createIntCounter(meter, PassedTestsCount, "Total number of passed tests")
+	skippedCounter := createIntCounter(meter, SkippedTestsCount, "Total number of skipped tests")
+	testsCounter := createIntCounter(meter, TotalTestsCount, "Total number of executed tests")
+
+	ctx, outerSpan := tracer.Start(ctx, "junit2otlp", trace.WithAttributes(runtimeAttributes...))
```

Comment:
NIT:

Can this be customised with something like `--name`?

Reply:
Or `OTEL_SERVICE_NAME` overrides it?

Reply:
I think this is a good feature request. Let me open an issue for that.

Reply:
https://github.com/mdelapenya/junit2otlp/issues/8